common/src/main/java/com/adaptivescale/rosetta/common/models/BaseModel.java

```diff
@@ -0,0 +1,28 @@
+package com.adaptivescale.rosetta.common.models;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class BaseModel {
+    private Map<String, Object> additionalProperties = new HashMap<>();
```

Comment:
Do we know what happens for other DB Types? This attribute is not displayed at all right?

Reply:
I have modified the code and now it will not displayed for other DB types.